openshift_images/create-images.adoc

```diff
@@ -1,18 +1,12 @@
-:context: create-images
 [id="creating_images"]
 = Creating images
 include::modules/common-attributes.adoc[]
+:context: create-images
 
-:toc: macro
-:toc-title:
-:prewrap!:
 toc::[]
 
 Learn how to create your own container images, based on pre-built images that are ready to help you.
-The process includes learning best practices for writing images,
-defining metadata for images, testing images and using a custom builder workflow to create images
- that can be used on
-{product-title}.
+The process includes learning best practices for writing images, defining metadata for images, testing images and using a custom builder workflow to create images that can be used on {product-title}.
```

Comment:
s/testing images/testing images, 

maybe s/images that can be used on/images to use with